crates/ty_python_semantic/Cargo.toml

```diff
@@ -68,8 +68,8 @@ glob = { workspace = true }
 indoc = { workspace = true }
 insta = { workspace = true }
 pretty_assertions = { workspace = true }
-quickcheck = { version = "1.0.3", default-features = false }
-quickcheck_macros = { version = "1.0.0" }
+quickcheck = { workspace = true }
+quickcheck_macros = { workspace = true}
```

Comment:
Huh, TIL Renovate can lift into the workspace level metadata like this.

Reply:
nah, that was me in https://github.com/astral-sh/ruff/pull/22247/commits/0853751e53e02a7b88280d198874b8db72783e0e :-) I pushed to the PR branch because we never add pins to crate-level `Cargo.toml` files -- we always declare dependencies as workspace dependencies and put the pins in the workspace `Cargo.toml` file